client/rest-client/src/main/java/org/infinispan/client/rest/impl/okhttp/RestCacheClientOkHttp.java

```diff
@@ -370,6 +371,13 @@ public CompletionStage<RestResponse> stats() {
       return client.execute(builder);
    }
 
+   @Override
+   public CompletionStage<RestResponse> statsReset() {
+      Request.Builder builder = new Request.Builder();
+      builder.method("POST",new FormBody.Builder().build()).url(cacheUrl + "?action=stats-reset").get();
```

Comment:
The `.get()` at the end is changing the request type.

```suggestion
      builder.method("POST",new FormBody.Builder().build()).url(cacheUrl + "?action=stats-reset");
```